src/coreclr/scripts/superpmi.py

```diff
@@ -2772,9 +2749,14 @@ def verify_replay_common_args():
                             "Unable to set pmi")
 
         coreclr_args.verify(args,
-                            "pmi_assemblies",
-                            lambda items: args.pmi is False or len(items) > 0,
-                            "Unable to set pmi_assemblies",
+                            "crossgen",
+                            lambda unused: True,
+                            "Unable to set crossgen")
+
+        coreclr_args.verify(args,
+                            "assemblies",
+                            lambda items: ((args.pmi is False) and (args.crossgen is False)) or len(items) > 0,
```

Comment:
What is the purpose of `(args.pmi is False) and (args.crossgen is False))`?

Reply:
I guess it would be more accurate to say:

`(((args.pmi is True) or (args.crossgen is True)) and len(items) > 0) || (((args.pmi is False) and (args.crossgen is False)) and len(items) == 0)`

Reply:
Basically, if `--pmi` or `--crossgen` is used, you have to specify `-assemblies`. If neither is used, you can't specify `-assemblies`.

Reply:
We can just make `--pmi` as default and simplify some logic here?